frontend/src/metabase/visualizations/echarts/cartesian/option/series.ts

```diff
@@ -99,6 +100,7 @@ const buildEChartsLineAreaSeries = (
     settings["stackable.stack_type"] != null ? `area_${yAxisIndex}` : undefined;
 
   return {
+    id: seriesModel.dataKey,
```

Comment:
Will we need to add these to the scatter plot series as well?

Reply:
Yep, it will be needed for hovers on scatter charts with breakouts